public/gatk-engine/src/main/java/org/broadinstitute/gatk/engine/CommandLineExecutable.java

```diff
@@ -72,6 +73,10 @@
 
     protected static Logger logger = Logger.getLogger(CommandLineExecutable.class);
 
+    static {
+        SRAAccession.setAppVersionString("GATK " + getVersionNumber());
```

Comment:
Supply a comment/javadoc explaining that this is setting the GATK version and why.